### PR TITLE
changed statut_segment_val table PK from code to id

### PR DESCRIPTION
**Pull request by Enola-sengeissen**

### CODE REVIEW

File: veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql [hunk 1]
```diff
@@ -158,9 +158,14 @@ ALTER TABLE ONLY veloroutes.segment
...
--- segment avancement
+-- statut_segment_val statut_segment_val_pkey
+ALTER TABLE ONLY veloroutes.statut_segment_val
+    ADD CONSTRAINT statut_segment_val_pkey PRIMARY KEY (id);
+
+
+-- segment avacement
 ALTER TABLE ONLY veloroutes.segment
-    ADD CONSTRAINT avancement FOREIGN KEY (avancement) REFERENCES veloroutes.etat_avancement_val(code);
+    ADD CONSTRAINT avacement FOREIGN KEY (avancement) REFERENCES veloroutes.etat_avancement_val(code) NOT VALID;
```
Comment (Gustry) [inline veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql:168]: Typo

Comment (rldhont) [inline veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql:168]:
```suggestion
    ADD CONSTRAINT avancement FOREIGN KEY (avancement) REFERENCES veloroutes.etat_avancement_val(code) NOT VALID;
```

File: veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql [hunk 2]
```diff
@@ -23,6 +23,16 @@ ALTER TABLE ONLY veloroutes.niveau_administratif_val
     ADD CONSTRAINT "FKcode" UNIQUE (code);
 
 
+-- etat_avancement_val avancement_PK
+ALTER TABLE ONLY veloroutes.etat_avancement_val
+    ADD CONSTRAINT "avancement_PK" PRIMARY KEY (id);
```
Comment (rldhont) [inline veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql:28]:
```suggestion
ALTER TABLE ONLY veloroutes.etat_avancement_val
    ADD CONSTRAINT etat_avancement_val_pkey PRIMARY KEY (id);
```

File: veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql [hunk 3]
```diff
@@ -53,11 +63,6 @@ ALTER TABLE ONLY veloroutes.revetement_val
...
 
 
--- statut_segment_val code8
-ALTER TABLE ONLY veloroutes.statut_segment_val
-    ADD CONSTRAINT code8 PRIMARY KEY (code);
-
-
 -- etat_avancement_val codeFK
 ALTER TABLE ONLY veloroutes.etat_avancement_val
     ADD CONSTRAINT "codeFK" UNIQUE (code);
```
Comment (rldhont) [inline veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql:68]:
```suggestion
ALTER TABLE ONLY veloroutes.etat_avancement_val
    ADD CONSTRAINT code9 UNIQUE (code);
```

Comment (Enola-sengeissen) [reply veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql]: rectifications done in the following commit

**Pull request merged**